ReactiveUI.XamForms/XamForms/ActivationForViewFetcher.cs

```diff
@@ -17,6 +14,7 @@ public int GetAffinityForView(Type view)
         {
             return
                 (typeof(Page).GetTypeInfo().IsAssignableFrom(view.GetTypeInfo())) ||
+                (typeof(View).GetTypeInfo().IsAssignableFrom(view.GetTypeInfo())) ||
```

Comment:
Maybe splitting this type in 3 activators (one for each type) will avoid the growing complexity of the below method and clearly separate each behavior.